packages/webpack-plugin/src/load-stylable-config.ts

```diff
@@ -0,0 +1,26 @@
+import { Compiler } from 'webpack';
+import findConfig from 'find-config';
```

Comment:
if the webpack plugin already uses `@file-services/node`, it has a method for that (and maybe we can avoid the dep)